drivers/gpu/drm/vc4/vc4_dsi.c

```diff
@@ -306,11 +306,11 @@
 # define DSI0_PHY_AFEC0_RESET			BIT(11)
 # define DSI1_PHY_AFEC0_PD_BG			BIT(11)
 # define DSI0_PHY_AFEC0_PD			BIT(10)
-# define DSI1_PHY_AFEC0_PD_DLANE3		BIT(10)
+# define DSI1_PHY_AFEC0_PD_DLANE3		BIT(8)
```

Comment:
Interleaving the bit positions for two registers is confusing enough, but including some of them out of order makes it worse, even if it hides the inconsistency of the bit assignments.

Reply:
It's an upstream file. Whilst it should get upstreamed, how many diffs do you wish to maintain?

Reply:
Your change isn't upstream.

Reply:
In other words, move the renumbered definitions into the correct position in the weird, interleaved sequence.

Reply:
OK. Your initial comment seemed to imply you wanted the interleaving removed, which would have made it quite a large downstream patch.
Done.